### PR TITLE
While typing numbers, don't convert . (dot) or : (colon)

### PR DESCRIPTION
**Pull request by mominul**
This fixes following issues
- [x] While typing numbers, pressing . should make it . not । (#13)
- [x] While typing numbers, pressing : should make it : not ঃ (#14)

### CODE REVIEW

**Comment by mominul** [comment]
Oops, I made the pull request with master branch... :grin:

**Comment by torifat** [comment]
@mominul Thanks.

File: AvroParser.m [hunk 1]
```diff
@@ -52,22 +52,23 @@ - (id)init {
...
             NSDictionary *jsonArray = [NSJSONSerialization JSONObjectWithData: jsonData options: NSJSONReadingMutableContainers error: &error];
-            
+
             if (!jsonArray) {
                 @throw error;
                 // @throw [NSException exceptionWithName:@"AvroParser init" reason:@"Error parsing JSON" userInfo:nil];
             } else {
                 _vowel = [[NSString alloc] initWithString:[jsonArray objectForKey:@"vowel"]];
                 _consonant = [[NSString alloc] initWithString:[jsonArray objectForKey:@"consonant"]];
+								_number = = [[NSString alloc] initWithString:[jsonArray objectForKey:@"number"]];
```
Comment (torifat) [inline AvroParser.m]: `= =` typo I guess.

Comment (mominul) [inline AvroParser.m]: Hmm, Atom was crazy with files :wink:

Comment (mominul) [inline AvroParser.m]: Oops, I haven't saw that `= =` and its an actually typo! :wink:

**Pull request merged**